central/xlsx/transform.go

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+
+package xlsx
+
+import (
+	"io"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+
+	"github.com/xuri/excelize/v2"
+)
+
+type mappingField struct {
+	defaultValue string // the default value defined in the MappingTable sheet
+	path         string // the path value defined in the MappingTable sheet
+}
+
+// ConvertXlsx transforms the xlsx file to the Converter interface
+func ConvertXlsx(file io.Reader, dto any) (Converter, error) {
+	f, err := excelize.OpenReader(file)
```

Comment:
before opening the file, add a quick validation in the beginning to filter the acceptable dto type, and the covnertor can be create here, too
```
switch v := dto.(type) {
case []*dto.Device:
  debug log
  create Device convertor
case []*dto.DeviceProfile:
  debug log
  create Profile convertor
default:
  print err
  return err
}
```

Reply:
Done